### DISCUSSION TITLE
Windows 11 H2 Update today ... reinstalled EP ... Quick Launch toolbar no longer able to be added!

### DISCUSSION THREAD

**Discussion by MylesMegee** [valinet/ExplorerPatcher#4051] [General]
I attempted to installed a major Win 11 update today. It would not proceed without uninstalling EP, which I did, thinking I could reinstall it again. 

After installing the latest version of EP that I was able to find, I am no longer able to add the QuickLaunch toolbar.

Is there a fix for this?

**Comment by TCHarp01** [comment]
It sounds as if you did what I did.  Unfortunately, I have not found a way to re-install EP after the 24H2 update, which means I am also without my Quick Launch toolbar and I am pretty upset about it.

**Reply by Amrsatrio** [reply]
Of course if you try to install anything pre-66.2 you will not be able to install it on 24H2, in other words, blocked by Microsoft. Install 67.1 and it works on 24H2. Don't forget to change your taskbar style to Windows 10 (ExplorerPatcher) manually.

**Reply by TCHarp01** [reply]
Yay!  Everything is now working.  67.1 was the key.  You guys are the absolute best.

**Comment by MylesMegee** [comment]
I am back to my Win 10 computer right now, but I found a workaround so that
you can pin the same shortcuts (with customizable icons) to the taskbar
in Win 11. It requires adding the program location to the Target. So to
create a shortcut to a website, you would add Chrome (or probably MS Edge)
before the website you want to access.
"C:\Program Files\Google\Chrome\Application\chrome.exe" https://onelook.com/

To create a shortcut to a PDF file, add whatever PDF app you use before the
file location:
"C:\Program Files (x86)\Adobe\Acrobat 2015\Acrobat\Acrobat.exe"
"C:\Users\YOURUSERNAME\Documents\PDFFILENAME.pdf"

To create a shortcut to a Word document, add the path for whatever version
of MS Office (OpenOffice) before the file location:
"C:\Program Files (x86)\Microsoft Office\Office10\WINWORD.EXE"
"C:\Users\YOURUSERNAME\Documents\WORDFILENAME.docx"

Unfortunately, the taskbar icons are a little smaller and there is more
space between them than with QuickLaunch, but it's functional and efficient
again. With Win 10, there is a way to make the icons smaller (Use small
taskbar buttons - pic below) but not larger unless you increase the scale
percentage in display settings under "Scale and layout." To then decrease
the size of desktop icons, press Ctrl while scrolling your mouse wheel
downward or press Ctrl Alt - (minus).
[image: image.png]
[image: image.png]

[image: image.png]

**Comment by pyrates999** [comment]
Open up EP Properties application, go to taskbar, and set **taskbar style** to **Windows 10 (ExplorerPatcher)**, _not_ **Windows 10**.  We no longer offer more support on the stock Windows 10 taskbar.

please see this for more info: https://github.com/valinet/ExplorerPatcher/wiki/ExplorerPatcher's-taskbar-implementation

**Reply by pageben** [reply]
Thank you!  I just came to the discussion to figure out the same issue and this got my taskbar looking the way I prefer.  I previously didn't realize I just had to click on the text that says "Taskbar Style *: Windows 10" -- It had a warning "Windows 10 taskbar is not available in this version of Windows."

Switching to "Windows 10 (ExplorerPatcher)" fixed it.

**Comment by peapanco** [comment]


**Reply by Amrsatrio** [reply]
User error. You downloaded the arm64 version or your x64 (ep_setup.exe) download is corrupt.
